features/lwipstack/lwip/src/include/lwip/lwip_errno.h

```diff
@@ -44,6 +44,7 @@ extern "C" {
 #endif
 
 #ifdef LWIP_PROVIDE_ERRNO
+#if LWIP_PROVIDE_ERRNO == 1
```

Comment:
line 46 and 47 are very similar, shouldn't ifdef be removed?

Reply:
The flag might be defined null